spec/Section 4 -- Introspection.md

```diff
@@ -256,30 +279,30 @@ Fields
 * All other fields must return {null}.
 
 
-#### Object
+**Object**
 
 Object types represent concrete instantiations of sets of fields. The
 introspection types (e.g. `__Type`, `__Field`, etc) are examples of objects.
 
-Fields
+Fields\:
 
 * `kind` must return `__TypeKind.OBJECT`.
 * `name` must return a String.
 * `description` may return a String or {null}.
-* `fields`: The set of fields that can be selected for this type.
+* `fields` must return the set of fields that can be selected for this type.
   * Accepts the argument `includeDeprecated` which defaults to {false}. If
     {true}, deprecated fields are also returned.
-* `interfaces`: The set of interfaces that an object implements.
+* `interfaces` must return the set of interfaces that an object implements.
```

Comment:
Is it worth noting that if the object doesn't implement any interfaces it must return the empty set (rather than `null`) or do you think this is sufficiently implied?

```suggestion
* `interfaces` must return the set of interfaces that an object implements 
  (if none, `interfaces` must return the empty set).
```